### PR TITLE
Update js_init.md

### PR DESCRIPTION
**Pull request by saphaljha**
## Purpose of this pull request

This pull request (PR) ...

## Affected DevDocs pages

-  ...

## Links to Magento source code

-  ...

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/guides/v2.3/javascript-dev-guide/javascript/js_init.md [hunk 1]
```diff
@@ -135,7 +135,7 @@ require([
 
 ## Calling JS components requiring initialization in JS files {#js_widget_init}
 
-To call a widget in JS code, use a notation similar to the following ([accordion]({{ page.baseurl }}/javascript-dev-guide/widgets/widget_accordion.html) widget is intiialized on the `[data-role=example]` element as illustration):
+To call a widget in JS code, use a notation similar to the following ([accordion]({{ page.baseurl }}/javascript-dev-guide/widgets/widget_accordion.html) widget is intialized on the `[data-role=example]` element as illustration):
```
Comment (rogyar) [inline src/guides/v2.3/javascript-dev-guide/javascript/js_init.md]: Hi @saphaljha. I'm still not sure that the spelling is correct
```suggestion
To call a widget in JS code, use a notation similar to the following ([accordion]({{ page.baseurl }}/javascript-dev-guide/widgets/widget_accordion.html) widget is initialized on the `[data-role=example]` element as illustration):
```

**Comment by dobooth** [comment]
Hi @saphaljha Thanks for the contribution. If you can please sign the Adobe CLA below, we can get this PR processed.

**Comment by dobooth** [comment]
Closing as abandoned. Moved fix to another PR.

**Pull request closed without merging**